engine/pyproject.toml

```diff
@@ -79,6 +79,7 @@ module = [
     "twilio.*",
     "uwsgidecorators.*",
     "whitenoise.*",
+    "uwsgi.*",
```

Comment:
we'll address the `uwsgi` and `celery.utils.debug` module `ignore_missing_imports` stuff at a later point alongside all of these other modules